blog/tags.yml

```diff
@@ -1,7 +1,7 @@
 fishjam-cloud:
```

Comment:
**question:** what ab this?

Reply:
blog doesn't work anyway so I won't be coming back to this since automerge did its thing